app/views/shared/partials/_device_problems.html.erb

```diff
@@ -0,0 +1,68 @@
+              <%= render partial: 'shared/icons/device-flow/ill-cleaning-keurig' %>
+              <p>Especially when they talk about “descaling.” Sounds dragon-y. Actually, all you have to do is fill the reservoir with one part white vinegar to two parts water, then run it until the reservoir is empty. Repeat every three to six months (translation: every year).</p>
+            </div>
+          </article>
+        </li>
+      <% end %>
+    </ul>
+
+    <ul class="problem-columns">
+        <% device.device_problems[split..(device_problem_count-1)].each_with_index do |problem, problem_index| %>
+        <li>
+          <article>
+            <header>
+              <h3><%= problem.statement %></h3>
+              <button aria-controls="problem-solution-<%= problem_index+1 %>" aria-expanded="false">
+                <span class="__inactive-list-item">Expand</span>
+                <%= image_tag 'icon-caret-hover.svg', alt: "" %>
+              </button>
+            </header>
+            <div class="problem__solutions __inactive-list-item problem-solution-<%= problem_index+1 %>" id="problem-solution-<%= problem_index+1 %>">
+              <%= render partial: 'shared/icons/device-flow/ill-cleaning-keurig' %>
+              <p>Especially when they talk about “descaling.” Sounds dragon-y. Actually, all you have to do is fill the reservoir with one part white vinegar to two parts water, then run it until the reservoir is empty. Repeat every three to six months (translation: every year).</p>
+            </div>
+          </article>
+        </li>
+        <% end %>
+    </ul>
+  </section>
+
+  <% else %>
+  <section class="device__problems device-<%= device.position %>__problems <%= "__inactive" if device.position != 1 %>">
+    <ul>
+      <% device.device_problems.each_with_index do |problem, problem_index| %>
+        <li>
+          <article>
+            <header>
+              <h3><%= problem.statement %></h3>
+              <button aria-controls="problem-solution-<%= problem_index+1 %>" aria-expanded="false">
+                <span class="__inactive-list-item">Expand</span>
+                <%= image_tag("icon-caret-hover.svg", :alt => "")%>
+              </button>
+            </header>
+            <div class="problem__solutions __inactive-list-item problem-solution-<%= problem_index+1 %>" id="problem-solution-<%= problem_index+1 %>">
+              <%= render partial: 'shared/icons/device-flow/ill-cleaning-keurig' %>
+              <p>Especially when they talk about “descaling.” Sounds dragon-y. Actually, all you have to do is fill the reservoir with one part white vinegar to two parts water, then run it until the reservoir is empty. Repeat every three to six months (translation: every year).</p>
+            </div>
+          </article>
+        </li>
+      <% end %>
+    </ul>
+  </section>
+<% end %>
```

Comment:
There are ways you can do the one-column/two-column logic without repeating code quite so much, but admittedly that is a little harder to figure out....  I might show you another time